README.md

```diff
@@ -63,6 +63,12 @@
 ## Stage 7
 <p>Continue working on GenePool, improve tests for Mutator and Evaluator dependencies and make first implementation of GenePool.</p>
 
+## Stage 8
```

Comment:
It is not a big mistake, but updating Stage9 includes Stage 8 details, so other PR is reduntant. If you wanted to have them indepently, Stage8 should be empty here.

Reply:
you have right :)